x-pack/plugins/alerting/server/init.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Legacy } from 'kibana';
+import {
+  createAlertRoute,
+  deleteAlertRoute,
+  findRoute,
+  getRoute,
+  listAlertTypesRoute,
+  updateAlertRoute,
+} from './routes';
+import { AlertingPlugin, Services } from './types';
+import { AlertTypeRegistry } from './alert_type_registry';
+import { AlertsClient } from './alerts_client';
+
+export function init(server: Legacy.Server) {
+  const { callWithInternalUser } = server.plugins.elasticsearch.getCluster('admin');
+  const savedObjectsClientWithInternalUser = server.savedObjects.getSavedObjectsRepository(
+    callWithInternalUser
+  );
+
+  const services: Services = {
+    log: server.log,
+    callCluster: callWithInternalUser,
+    savedObjectsClient: savedObjectsClientWithInternalUser,
```

Comment:
Instead of passing the repository with that is bound to the internal server user, can we use an instance of the SavedObjectsClient that isn't bound to any authorization headers, until we have long running tokens?